src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1724,8 +1724,11 @@ private void DoRunspaceInitialization(bool importSystemModules, bool skipProfile
                 if (_configuration != null)
                     shellId = _configuration.ShellId;
                 else
+#if CORECLR                
+                    shellId = Platform.IsInbox ?  "Microsoft.PowerShell" : "PowerShell";
+#else
```

Comment:
@charub the discussion conclusion is that we don't need to change the profile file names for powershell core. Please see the comments in #1190 for details.
Give the discussion conclusion, do you think we need to change anything to resolve issue #1190 ?